### PR TITLE
python@3.11 3.11.8

### PR DESCRIPTION
**Pull request by branchv**
Created with `brew bump-formula-pr`.

- [x] `resource` blocks have been checked for updates.

### CODE REVIEW

**Review by chenrui333 — APPROVED** [review]

**Comment by branchvincent** [comment]
Some `swift` and `root` failures so far. Need another run for self-hosted linux

**Comment by ZhongRuoyu** [comment]
Linux self-hosted has too many pending jobs. Will do a rerun later.

File: Formula/p/python@3.11.rb [hunk 1]
```diff
@@ -73,13 +72,13 @@ class PythonAT311 < Formula
...
-    url "https://files.pythonhosted.org/packages/1f/7f/4da15e07ccd11c84c1ccc8f6e24288d5e76c99441bf80e315b33542db951/pip-23.3.1.tar.gz"
-    sha256 "1fcaa041308d01f14575f6d0d2ea4b75a3e2871fe4f9c694976f908768e14174"
+    url "https://files.pythonhosted.org/packages/94/59/6638090c25e9bc4ce0c42817b5a234e183872a1129735a9330c472cc2056/pip-24.0.tar.gz"
+    sha256 "ea9bd1a847e8c5774a5777bb398c19e80bcd4e2aa16a4b301b718fe6f593aba2"
   end
 
   resource "setuptools" do
-    url "https://files.pythonhosted.org/packages/4b/d9/d0cf66484b7e28a9c42db7e3929caed46f8b80478cd8c9bd38b7be059150/setuptools-69.0.2.tar.gz"
-    sha256 "735896e78a4742605974de002ac60562d286fa8051a7e2299445e8e8fbb01aa6"
+    url "https://files.pythonhosted.org/packages/fc/c9/b146ca195403e0182a374e0ea4dbc69136bad3cd55bc293df496d625d0f7/setuptools-69.0.3.tar.gz"
```
Comment (bertonha) [inline Formula/p/python@3.11.rb:80]: maybe update to `setuptools` `69.1.0`

**Comment by p-linnane** [comment]
Removed self-hosted Linux label since we just need it for dep testing. `icu4c` is using that runner right now, so will readd the label for dep testing so we can get all the macOS runs out of the way.

**Comment by cho-m** [comment]
Migrating alias to `python@3.12` should have reduced some dependents (i.e. those using `uses_from_macos "python"`).

Removing some `python-` formulae should have reduced some more, but may not be enough to avoid self-hosted Linux.

**Comment by p-linnane** [comment]
`texlive` on it's own is our blocker with skipping self-hosted unfortunately.

**Comment by cho-m** [comment]
> `texlive` on it's own is our blocker with skipping self-hosted unfortunately.

I'm still curious if hardlink-ing files can bring down disk space during bottle unpack. I've often seen the first unpack to `/tmp` succeed and then the second `cp` to `HOMEBREW_PREFIX` fail.

---

Anyway, may need to check if there is a `python@3.12` blocker for `texlive`.

**Comment by p-linnane** [comment]
Good news is barring anything weird going on with the Linux dep run, we should be able to get this merged.

**Review by p-linnane — APPROVED** [review]

**Review by ZhongRuoyu — APPROVED** [review]

**Pull request merged**